week-5-practise/js/main.js

```diff
@@ -0,0 +1,66 @@
+//// PART-1 ////
+
+var placeToJumbotron = document.querySelector(".jumbotron");
+var placeToDonateABike = document.querySelector(".btn-primary"+".btn-lrg");
+var placeToVolunteer = document.querySelector(".btn-secondary");
+
+var blueButton = document.querySelector("#blueBtn");
+blueButton.addEventListener("click", blueButtonFunction);
+function blueButtonFunction() {
+    placeToJumbotron.style.backgroundColor = "#588fbd";
+    placeToDonateABike.style.backgroundColor = "#ffa500";
+    placeToVolunteer.style.backgroundColor = "black";
+    placeToVolunteer.style.color = "white";   
+}
+
+var orangeButton = document.querySelector("#orangeBtn");
+orangeButton.addEventListener("click", orangeButtonFunction);
+function orangeButtonFunction() {
+    placeToJumbotron.style.backgroundColor = "#f0ad4e";
+    placeToDonateABike.style.backgroundColor = "#5751fd";
+    placeToVolunteer.style.backgroundColor = "#31b0d5";
+    placeToVolunteer.style.color = "white";
+}
+
+var greenButton = document.querySelector("#greenBtn");
+greenButton.addEventListener("click", greenButtonFunction);
+function greenButtonFunction() {
+    placeToJumbotron.style.backgroundColor = "#87ca8a";
+    placeToDonateABike.style.backgroundColor = "black";
+    placeToVolunteer.style.backgroundColor = "#8c9c08";
+}
+
+////////// PART-2 /////
+
+var formTag = document.querySelector("form");
+formTag.lastElementChild.addEventListener("click", submitButtonFunction); 
+event.preventDefault();
```

Comment:
Did this give you a runtime error? I can't see `event` being declared anywhere. Let me know what you were trying to achieve with this line of code and I can help you out.

Reply:
I am asked that "In your function that handles clicks on the `Submit` button you will need to call `event.preventDefault()` to stop the browser from refreshing the page." in the question. That's why I have used it.

Reply:
In that case, you'll need to move it into the function. Outside the function, where it is now, you'll find that `event` is undefined.